test_scripts/Polices/External_UCS/017_ATF_P_TC_PTU_without_disallowed_by_external_consent_entities_off_struct.lua

```diff
@@ -0,0 +1,120 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PTU without "disallowed_by_external_consent_entities_off struct
+--
+-- Description:
+-- In case:
+-- SDL receives PolicyTableUpdate without “disallowed_by_external_consent_entities_off:
+-- [entityType: <Integer>, entityId: <Integer>]” -> of "<functional grouping>"
+-- -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PTU as valid (with the pre-conditions of all other valid PTU content)
+-- b. do not create this "disallowed_by_external_consent_entities_off" field
+-- of the corresponding "<functional grouping>" in the Policies database.
+--
+-- Preconditions:
+-- 1. Start SDL (make sure 'disallowed_by_external_consent_entities_off' section is omitted in PreloadedPT)
```

Comment:
@dboltovskyi, as I understand from requirement, you should have actions: 
1. Precondtion:
Sdl_preloaded_pt.json with disallowed_by_external_consent_entities_off
-> Check sdl_shapshot.json: disallowed_by_external_consent_entities_off exist
2. In ptu file do not include disallowed_by_external_consent_entities_off
3. Make PTU (SUCCESS)
-> Check sdl_shapshot.json: disallowed_by_external_consent_entities_off is removed.

Please clarify and if agree update the script in this order.